gradle.properties

```diff
@@ -14,12 +14,12 @@
 # limitations under the License.
 #
 
-GRADLE_PLUGIN_VERSION=3.1.2
-ANDROID_SUPPORT_VERSION=27.1.1
+GRADLE_PLUGIN_VERSION=3.1.4
+ANDROID_SUPPORT_VERSION=28.0.1
```

Comment:
This can probably be removed, I don't think that the support library version will ever be used after converting to Jetpack. 

I recommend adding an `ANDROID_X_VERSION=1.0.0-rc1` line though, and replacing all of the duplicate version definitions in the `build.gradle` files

Reply:
Done 😄